spec/Makefile

```diff
@@ -1,10 +1,11 @@
-.PHONY: all clean
+.PHONY: all clean diagrams
 
-all: index.html webgpu.idl
+all: index.html webgpu.idl diagrams
 
 clean:
 	rm -f index.html index.pre.html webgpu.idl
 
+# Generate spec HTML from Bikeshed source.
 index.pre.html: *.bs
```

Comment:
Does this need to depend on diagrams? (Is bikeshed inlining the SVGs into the spec? If so, won't this pickup the old version as you generate the new one after?)

Reply:
No, bikeshed is not inlining the SVGs. The `<object>` is preserved as is in the output HTML.

Reply:
Hence the (not very well advertised) comment:
> # Generate diagrams from source. You can rebuild these without rebuilding the spec.

Reply:
I don't know how external resources will interact with publishing on TR but I'm pretty sure they're somehow supported. If we really want to include them in the generated HTML I think we could do it with `<pre class=include>`.

Reply:
cc @tidoust

Reply:
Interesting, did you consider having bikeshed inline the SVGs to keep the spec self contained?

```
<pre class=include-raw>
path: myDiagram.svg
</pre>
```

(Though, it does mean the spec has to rebuild if the SVGs change)

Reply:
I confirm spec-prod will automatically copy the SVG files targeted by the `<object>` elements to /TR, so no need to inline them.